### PR TITLE
Ruby-Ecosystem translation to german

### PR DESCRIPTION
**Pull request by coastalchief**

### CODE REVIEW

**Comment by vendethiel** [comment]
Thanks!

**Comment by vendethiel** [comment]
Did you mean to commits scala and ruby as well?

**Comment by coastalchief** [comment]
Basically Yes!. I've merged the scala and ruby article in my master and created the pull-request then, the ruby-ecosystem came directly from a branch.

File: de-de/scala-de.html.markdown [hunk 1]
```diff
@@ -0,0 +1,810 @@
...
+
+// Eine Datei schreiben  
+```
+val writer = new PrintWriter("myfile.txt")
+writer.write("Writing line for line" + util.Properties.lineSeparator)
+writer.write("Another line here" + util.Properties.lineSeparator)
+writer.close()
+```
+
+## Further resources
```
Comment (vendethiel) [inline de-de/scala-de.html.markdown]: don't you want to translate that? (and add a "[EN]" flag)

Comment (coastalchief) [inline de-de/scala-de.html.markdown]: I've added german resources, translated the heading and marked EN resources as such.

Comment (vendethiel) [inline de-de/scala-de.html.markdown]: yeah, but you didn't translate the headline :P

**Pull request merged**